python/taichi/lang/__init__.py

```diff
@@ -247,18 +247,30 @@ def static_print(*args, __p=print, **kwargs):
     __p(*args, **kwargs)
 
 
-def benchmark(func, repeat=100, args=()):
+def benchmark(func, repeat=300, args=()):
     import taichi as ti
     import time
-    for i in range(repeat // 3):
-        func(*args)  # compile the kernel first
+    func(*args)  # compile the kernel first
```

Comment:
@yuanming-hu Why we needed to compile so many times?

Reply:
We need the first iter for compilation, and second & thrid iterations for warming up the instruction/data caches.

Reply:
I guess range(10) is enough? Also consider if repeat = 1?

Reply:
Anyway, let's add a `Discussion` mark here if you don't sure this work.